andela/ah-frontend-athena
# #162163232 enable user rate article

### What does this PR do?
Add rating of an article 

### Description of Task to be completed?
When a user views an article, they should be able to rate the article

### What are the relevant pivotal tracker stories?
#162163232]

### How to test the feature
- Click on the read more of any article
- The average rating of the article is displayed in form of star icons
- Click on the stars to give it a new rating


### Screenshots
![image](https://user-images.githubusercontent.com/10868919/51732685-1028ce80-2090-11e9-97a7-2d4d83b61b96.png)
